tests/unit/store/traction/pacbio/runs/actions.spec.js

```diff
@@ -197,8 +197,8 @@ describe('#buildWell', () => {
       binding_kit_box_barcode: '',
       pools: [],
       pre_extension_time: 2,
-      loading_target_p1_plus_p2: '',
-      use_adaptive_loading: 'False',
+      loading_target_p1_plus_p2: 0.85,
```

Comment:
Again here we are still sending use_adaptive_loading to the service. Not sure whether we need to?

Reply:
This is just setting the default attributes while creating/building a new well. It will not be sent back to the service.

Reply:
It is just assigning a default value when creating/building a new well. This value will not included in creating the request payload to the service.

Reply:
I understand. I think the point is that you do not need to it at all because it is a computed value based on the loading_target_p1_plus_p2 in the component. Unless I am missing something? More than happy to talk through it with you.

Reply:
Thank you for pointing it. Removed the lines.